src/main/java/mate/academy/model/Ticket.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private MovieSession movieSession;
```

Comment:
Let's add `@JoinColumn` with explicit name
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "movie_session_id)
    private MovieSession movieSession;
```